AY2122S2-CS2103T-T13-3/tp
# Use module index in Add and Edit meeting

## Related issues
Closes #79

## Description
- Add and Edit meetings now support the usage of `MODULE_INDEX`
- Refactored `AddMeetingCommand` to take in an `AddMeetingCommandDescriptor` instead
- Refactored `EditMeetingCommandDescriptor`
- Commented out broken test cases (Add, Edit meeting)

## How to test
1. Open the app. (Reset data file if needed)
2. Run the following command: `add n/PE dry run u/https://www.zoom.com d/01-04-2022 1400 dur/2 m/1 r/Y` 
3. The meeting should be created and tagged to the first module in the module list.
4. Run the following command: `edit 4 m/2` 
5. The meeting at index 4 should now be tagged to the second module in the module list.

## Other information
- The broken test cases have been commented out to allow the rest of the code to compile successfully. 
- Manual testing showed no functionality bugs 🤞 
- To easily refer back to it when fixing it in the future, they are included with a `// TODO MODULE INDEX: ...` comment. Use `Cmd/Ctrl + Shift + F` to search for these comments.
